env-secrets.template

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Thanks for including this - it helped make it _really_ easy to set up your Minikube